http.js

```diff
@@ -117,7 +117,19 @@ function start (entry, opts) {
     })
   })
 
-  router.route(/\/([a-zA-Z0-9-_.]+)\.js\?{0,1}[\w&=]*$/, function (req, res, params) {
+  router.route(/^\/assets\/([^?]*)(\?.*)?$/, function (req, res, params) {
```

Comment:
Dang yeah, we can't quite merge this without also supporting `content` - it'd otherwise cause a regression. Probably can be done a s a quick hack tho :sparkles:

Reply:
Oh lol, no so this is just us. LGTM!